posts/2018-05-15-Rust-turns-three.md

```diff
@@ -44,7 +44,7 @@ Finally, the Rust community continues to work on inclusivity, through outreach
 programs like [Rust Reach](https://blog.rust-lang.org/2018/04/02/Increasing-Rusts-Reach-2018.html) and
 [RustBridge](https://rustbridge.github.io/), as well as
 [structured mentoring](https://blog.rust-lang.org/2017/09/18/impl-future-for-rust.html) and
-investments in [documentation](https://rust-lang-nursery.github.io/rustc-guide/)
+investments in [documentation](https://rust-lang-nursery.github.io/rustc-dev-guide/)
```

Comment:
Shouldn't we use this opportunity to remove the redirection through the nursery?

Reply:
addressed in 4367fad9d2bc955da039db41efc61627a0984c46